ajax/libs/placeholder-shiv/package.json

```
@@ -12,7 +12,7 @@
     "url": "https://github.com/walterdavis/placeholder-shiv.git"
   },
   "license": {
-    "type": "BSD",
+    "type": "BSD 3-Clause",
```

```suggestion
    "type": "BSD-3-Clause",
```

SPDX identifier is `BSD-3-Clause` => https://spdx.org/licenses/BSD-3-Clause.html